packages/react/checkbox/src/Checkbox.test.tsx

```diff
@@ -47,52 +33,19 @@ describe('given a default Checkbox', () => {
     it('should render a visible indicator', () => {
       expect(indicator).toBeVisible();
     });
-
-    it('should have a radix attribute on the indicator', () => {
-      const partDataAttr = `data-${getSelector('CheckboxIndicator')}`;
-      expect(indicator).toHaveAttribute(partDataAttr);
-    });
-
-    it('should have a data-state attribute on the container set to `checked`', () => {
-      expect(checkbox).toHaveAttribute('data-state', 'checked');
-    });
-
-    it('should have a data-state attribute on the indicator set to `checked`', () => {
-      expect(indicator).toHaveAttribute('data-state', 'checked');
-    });
-
-    describe('and clicking the checkbox again', () => {
```

Comment:
Do we want to keep this one somehow because it's testing interaction?

Reply:
I'll check this now.

Reply:
Yeah good shout, I had remove it cause it was checking data-state, but I've rewritten to check for the indicator